lib/widgets/content.dart

```diff
@@ -180,30 +198,39 @@ class MessageImage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final message = InheritedMessage.of(context).message;
+
     // TODO multiple images in a row
     // TODO image hover animation
     final src = node.srcUrl;
 
     final store = PerAccountStoreWidget.of(context);
     final resolvedSrc = resolveUrl(src, store.account);
 
-    return Align(
-        alignment: Alignment.centerLeft,
-        child: Padding(
-            // TODO clean up this padding by imitating web less precisely;
-            //   in particular, avoid adding loose whitespace at end of message.
-            // The corresponding element on web has a 5px two-sided margin…
-            // and then a 1px transparent border all around.
-            padding: const EdgeInsets.fromLTRB(1, 1, 6, 6),
-            child: Container(
-                height: 100,
-                width: 150,
-                alignment: Alignment.center,
-                color: const Color.fromRGBO(0, 0, 0, 0.03),
-                child: RealmContentNetworkImage(
-                  resolvedSrc,
-                  filterQuality: FilterQuality.medium,
-                ))));
+    return GestureDetector(
+      onTap: () {
+        Navigator.of(context).push(getLightboxRoute(
+          context: context, message: message, src: resolvedSrc));
+      },
+      child: Align(
+          alignment: Alignment.centerLeft,
+          child: Padding(
+              // TODO clean up this padding by imitating web less precisely;
+              //   in particular, avoid adding loose whitespace at end of message.
+              // The corresponding element on web has a 5px two-sided margin…
+              // and then a 1px transparent border all around.
+              padding: const EdgeInsets.fromLTRB(1, 1, 6, 6),
+              child: Container(
+                  height: 100,
+                  width: 150,
+                  alignment: Alignment.center,
+                  color: const Color.fromRGBO(0, 0, 0, 0.03),
+                  child: LightboxHero(
+                    message: message,
+                    src: resolvedSrc,
+                    child: RealmContentNetworkImage(
+                      resolvedSrc,
```

Comment:
Hmm, this feels kind of redundant. Can the LightboxHero use `src` to construct this RealmContentNetworkImage widget for itself?